admin/views/test-developer/upload-new-form.ejs

```diff
@@ -67,13 +67,13 @@
 
         <h2 class="heading-medium">Upload file(s)</h2>
 
-        <form action="/test-developer/upload-new-form" method="post" enctype="multipart/form-data" id="upload-form">
+        <form action="/test-developer/upload-new-form" method="post" enctype="multipart/form-data" encoding="multipart/form-data" id="upload-form">
```

Comment:
@jon-shipley should we include the encoding attribute then in the rest of the areas where we upload forms?

Reply:
@GeorgeChatzigiannis - It's a specific hack for IE Edge.  It's not standard HTML.  It may help if the form doesn't upload in IE Edge.